src/store/modules/auth.js

```diff
@@ -0,0 +1,36 @@
+import { postUserLogin } from "@/services/HttpClientService";
+
+export default {
+  namespaced: true,
+  state: {
+    user: null,
+    token: null,
+  },
+  mutations: {
+    setUser(state, payload) {
+      state.user = payload.user
+      state.token = payload.token
+    },
+    clearUser(state) {
+      state.user = null
+      state.token = null
+    },
+  },
+  actions: {
+    async login({commit}, payload) {
+      const user = await postUserLogin(payload)
+      commit('setUser', user)
+      localStorage.setItem("userAuth", JSON.stringify(user.user));
```

Comment:
didn't we say we are going to use vuex instead of the localStorage?

Reply:
done